code/scripting/api/objs/hudconfig.cpp

```diff
@@ -343,6 +343,33 @@ ADE_VIRTVAR(UsesIffForColor,
 	return ADE_RETURN_TRUE;
 }
 
+ADE_VIRTVAR(isCustomGauge,
```

Comment:
Just wondering, would this make more sense as a function instead of a virtvar?

Reply:
The distinction between func and virtvar is pretty loose in our API but given this is a member property of a hud gauge, I would say no. IMO, functions should really be reserved for API methods that actually do things instead of just returning values.

Reply:
Sounds good, thanks!